rpm/libsailfishapp.spec

```diff
@@ -11,6 +11,7 @@ BuildRequires: pkgconfig(Qt5Qml)
 BuildRequires: pkgconfig(qdeclarative5-boostable)
 BuildRequires: pkgconfig(mlite5)
 BuildRequires: sailfish-qdoc-template
+Requires: qt5-qttools-linguist
```

Comment:
I wonder why this is placed here, when it should (according to the PR title and its use) be placed somewhere below line 25 "``"

Reply:
Oh, I pushed an old version and thinking that it was the wrong place, I changed it. But I forgot to save my changes before commiting. Thanks a lot for noticing.

Reply:
Thank you for resolving this!